src/config.ts

```diff
@@ -26,6 +26,9 @@ const SENDGRID_NEW_DEVELOPER_ACCOUNT_TEMPLATE_SOURCE = defineString(
 // Igbo API
 const ENV_MAIN_KEY = defineString('ENV_MAIN_KEY').value();
 
+// Nkọwa Okwu AI Models
+const ENV_IGBO_TO_ENGLISH_URL = defineString('ENV_IGBO_TO_ENGLISH_URL').value();
```

Comment:
i'm mainly leaving this note to remind us to assign this environment variable in the cloud function envs list on GCP

Reply:
done ✅